Flaky test: `chainer_tests/functions_tests/loss_tests/test_contrastive.py::TestContrastive`

https://jenkins.preferred.jp/job/chainer/job/chainer_pr/1513/TEST=CHAINERX_chainer-py3,label=mn1-p100/console

`tests.chainer_tests.functions_tests.loss_tests.test_contrastive.TestContrastive_param_11_{label_dtype=int32, backward_options={'rtol': 0.01, 'atol': 0.001}, dtype=float16, batchsize=5, forward_options={'rtol': 0.01, 'atol': 0.01}, reduce='no', margin=2, input_dim=3, double_backward_options={'rtol': 0.3, 'atol': 0.3}}.test_backward_gpu_no_cudnn
`
```
E           AssertionError: Parameterized test failed.
E           
E           Base test method: TestContrastive.test_backward_gpu_no_cudnn
E           Test parameters:
E             label_dtype: <class 'numpy.int32'>
E             backward_options: {'rtol': 0.01, 'atol': 0.001}
E             dtype: <class 'numpy.float16'>
E             batchsize: 5
E             forward_options: {'rtol': 0.01, 'atol': 0.01}
E             reduce: no
E             margin: 2
E             input_dim: 3
E             double_backward_options: {'rtol': 0.3, 'atol': 0.3}
E           
E           
E           (caused by)
E           AssertionError: check_backward failed (eps=0.001 atol=0.001 rtol=0.01)
E           inputs[0]:
E           [[-0.73339844  0.89257812  0.43701172]
E            [ 0.82275391  0.48852539 -0.03103638]
E            [ 0.73388672  0.99414062  0.77929688]
E            [-0.69433594 -0.28833008 -0.72753906]
E            [ 0.41308594  0.85644531  0.69824219]]
E           inputs[1]:
E           [[-0.99169922 -0.61132812  0.37646484]
E            [ 0.55859375 -0.81054688 -0.97412109]
E            [ 0.89697266 -0.08062744 -0.23132324]
E            [ 0.79492188 -0.88427734 -0.88574219]
E            [ 0.42871094 -0.03460693  0.19641113]]
E           inputs[2]:
E           [0 0 0 1 0]
E           grad_outputs[0]:
E           [ 0.66015625  0.99023438  0.99169922  0.36303711  0.84179688]
E           directions[0]:
E           [[ 0.0854787   0.14562142 -0.31627069]
E            [ 0.09447964  0.08634321  0.15309343]
E            [ 0.06313489 -0.31290636 -0.1966178 ]
E            [-0.00190714  0.08900132 -0.42851539]
E            [ 0.08677065  0.16406408 -0.25193511]]
E           directions[1]:
E           [[ 0.39248452  0.05245816  0.03548756]
E            [ 0.16464156  0.0719736   0.00598769]
E            [ 0.11164177  0.25904396  0.16609268]
E            [-0.09428912  0.03132483  0.01150929]
E            [ 0.02805343 -0.11975318 -0.28098777]]
E           gradients (numeric):  0.01864469597545154
E           gradients (backward): 0.017391558538361745
E           
E           x: numeric gradient, y: backward gradient
E           Not equal to tolerance rtol=0.01, atol=0.001
E           
E           (mismatch 100.0%)
E            x: array(0.01864469597545154)
E            y: array(0.017391558538361745)
E           
E           assert_allclose failed: 
E             shape: () ()
E             dtype: float64 float64
E             i: (0,)
E             x[i]: 0.01864469597545154
E             y[i]: 0.017391558538361745
E             relative error[i]: 0.07205434948947573
E             absolute error[i]: 0.0012531374370897928
E             relative tolerance * |y[i]|: 0.00017391558538361747
E             absolute tolerance: 0.001
E             total tolerance: 0.0011739155853836174
E           x: 0.01864469597545154
E           y: 0.017391558538361745
```

--- a/tests/chainer_tests/functions_tests/loss_tests/test_contrastive.py
+++ b/tests/chainer_tests/functions_tests/loss_tests/test_contrastive.py
@@ -15,7 +15,7 @@ from chainer.testing import attr
 @testing.parameterize(*testing.product_dict(
     [{'dtype': numpy.float16,
       'forward_options': {'rtol': 1e-2, 'atol': 1e-2},
-      'backward_options': {'rtol': 1e-2, 'atol': 1e-3},
+      'backward_options': {'rtol': 1e-2, 'atol': 1e-2},
       'double_backward_options': {'rtol': 3e-1, 'atol': 3e-1}},
      {'dtype': numpy.float32,
       'forward_options': {'rtol': 1e-2},

